sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentController.java

```diff
@@ -1,29 +1,47 @@
 package uk.ac.ebi.atlas.experimentpage;
 
-import com.google.gson.Gson;
-import com.google.gson.JsonArray;
-import com.google.gson.JsonObject;
+import com.google.gson.*;
```

Comment:
Try not to use wildcards in import statements. This is probably IDEA’s fault, go to *Preferences* > *Editor* > *Code Style* > *Java* and set a ridiculous high number to replace import statements with asterisks.

![image](https://user-images.githubusercontent.com/4425744/38624251-91e62b72-3d9f-11e8-9480-de78a210499b.png)

Reply:
Yes, it was IntelliJ doing it. I tried to fight it manually but I lost. Will change the settings.